docs/user_guide/discretisation/IncreasingWidthDiscretiser.rst

```diff
@@ -0,0 +1,134 @@
+.. _increasing_width_discretiser:
+
+.. currentmodule:: feature_engine.discretisation
+
+IncreasingWidthDiscretiser
+==========================
+
+The :class:`IncreasingWidthDiscretiser()` divides continuous numerical variables into
+intervals of increasing width with equal increments. Note that the
+proportion of observations per interval may vary.
+
+This discretisation technique is great when the distribution of the variable is right skewed.
+
+The :class:`IncreasingWidthDiscretiser()` works only with numerical variables. A list of
+variables to discretise can be indicated, or the discretiser will automatically select
+all numerical variables in the train set.
+
+**Note**
+
+This discretiser could be numerically unstable when the values of the variable are small (their order of magnitude is small).
```

Comment:
You are looking into this, correct? It'd be great to say when it would be unstable and when one can safely use it.

Reply:
Yep. In my code in the project I use np.digitize and sklearn logic from KBinsDiscretizer and it works fine. So I have my suspicions about pandas

Reply:
Fixed